src/environments/environment.prod.ts

```diff
@@ -11,4 +11,6 @@ export const environment = {
   DOCUMENT_RETRIEVAL_API_KEY: '',
   FEEDBACK_URI: '',
   SENTRY_DSN: '',
+  VTM_GTM_CONTAINER_ID: '',
+  VTM_GTM_MEASUREMENT_ID: '',
```

Comment:
Have you spoke to platform and get this added in there too? Not sure if you are aware of that step

Reply:
Needs to be for all non dev environments too

Reply:
Yeah dont worry there is another task which is in progress for G-Force to add the properties into AWS Secrets

Reply:
This is for all envs too